### DISCUSSION TITLE
Issue with Splats from Luma

### DISCUSSION THREAD

**Discussion by CiberNin** [playcanvas/supersplat#138] [General]
[Boxen_point_cloud.zip](https://github.com/user-attachments/files/16312911/Boxen_point_cloud.zip)
I don't know if Luma changed something about how they encode or yall' changed how you read, but I get this error in console when loading the attached file.
`TypeError: Cannot read properties of undefined (reading '0')
    at bc.transform (gsplat-data.js:88:13)
    at ply.js:345:18`

**Comment by slimbuck** [comment]
Hi @CiberNin,

This file doesn't contain gaussian splat data. It contains only positions and colors. Could you have used the wrong Luma export option?

Supersplat could really do with a better error message here.

**Reply by CiberNin** [reply]
Oh yes my bad. Its been awhile since I used the software and I got mixed up.

**Comment by slimbuck** [comment]
No probs at all. We've created an issue to address the non error message in these cases.

https://github.com/playcanvas/supersplat/issues/139
